charts/error-pages/README.md

```diff
@@ -0,0 +1,56 @@
+# Global Error Pages
+
+## Introduction
+
+This chart installs a global error controller for an [error page for traefik](https://docs.traefik.io/configuration/backends/kubernetes/#global-default-backend-ingresses).
```

Comment:
Maybe re-phrase this, proposal:
This chart installs a controller for [Traefik](https://traefik.io) global [error pages]((https://docs.traefik.io/configuration/backends/kubernetes/#global-default-backend-ingresses).